src/ccaligner.cpp

```diff
@@ -33,62 +34,36 @@ void printHeader(std::string version)
     std::cout<<"=====================================================\n\n";
 }
 
-void printFooter()
+static void printFooter()
 {
     std::cout<<"\n\n=====================================================\n";
     std::cout<<"Issues? Open a ticket here\n";
     std::cout<<"https://github.com/saurabhshri/CCAligner/issues\n";
 }
 
-CCAligner::CCAligner(Params * parameters)
+int main(int argc, char *argv[])
 {
-    _parameters = parameters;
-    should_log = _parameters->verbosity ;
-}
+    printHeader("0.03 Alpha [Shubham]");
 
-int CCAligner::initAligner()
-{
-    if(_parameters->chosenAlignerType == approxAligner)
+    Params parameters;
+    parameters.inputParams(argc,argv);
+
+    if(parameters.chosenAlignerType == approxAligner)
     {
-        ApproxAligner * aligner = new ApproxAligner(_parameters->subtitleFileName, srt);
-        aligner->align();
-        delete aligner;
+        ApproxAligner(parameters.subtitleFileName, srt).align();
     }
 
-    else if(_parameters->chosenAlignerType == asrAligner)
+    else if(parameters.chosenAlignerType == asrAligner)
     {
-        PocketsphinxAligner * aligner = new PocketsphinxAligner(_parameters);
-        aligner->align();
-        //aligner->printAligned("Manual_Printing.json", json);
-        delete(aligner);
+        PocketsphinxAligner(&parameters).align();
     }
 
     else
     {
         FATAL(EXIT_INVALID_PARAMETERS, "Unsupported Aligner Type!");
+        return -1;
```

Comment:
This is unnecessary, as FATAL already returns an error code based on supplied parameter (here : EXIT_INVALID_PARAMETERS).